web/styles/compose.css

```diff
@@ -661,21 +662,19 @@ input.recipient_box {
     }
 
     .compose_control_button {
-        padding: 5px;
+        padding: 0.7px 5px 0;
```

Comment:
Hmmmmm. Where's this `0.7px` value coming from? I suspect you might be trying to eyeball the old design, but we might want to look and see whether a whole value (`1px`) or a zero value works just as well--even if it slightly differs from the old design. This could be the kind of thing worth discussing on CZO.